armci/src/common/capi.c

```diff
@@ -208,15 +208,6 @@ int ARMCI_Init_mpi_comm(MPI_Comm comm)
 #endif
 
 
-#if HAVE_SYS_WEAK_ALIAS_PRAGMA
```

Comment:
are you sure this is should be part of this commit?  it is unrelated.

Reply:
Commit was related to Issue #292